apps/common-app/src/examples/RuntimeTests/tests/animations/withTiming/arrays.test.tsx

```diff
@@ -102,17 +101,19 @@ describe('withTiming animation of ARRAY', () => {
       );
       const components = Object.values(COMPONENT_REF).map(refName => getTestComponent(refName));
 
+      // eslint-disable-next-line @typescript-eslint/no-misused-promises
       components.forEach(async (component, index) => {
         expect(await component.getAnimatedStyle('width')).toBe(
           startWidths[index] * scalars[index],
           ComparisonMode.DISTANCE,
         );
       });
       await wait(speed + 200);
-
-      components.forEach(async (component, index) => {
+      let index = 0;
+      for (const component of components) {
         expect(await component.getAnimatedStyle('width')).toBe(finalWidths[index], ComparisonMode.DISTANCE);
-      });
+        index += 1;
+      }
```

Comment:
Why we need this change?

Reply:
The syntax `array.foreach(async () => {})` is not allowed, one of eslint rules ban it.
After reading [this page](https://gist.github.com/joeytwiddle/37d2085425c049629b80956d3c618971) I got convincded that `for` loop is a valid way to write this code.